_tests_/pet.test.js

```diff
@@ -0,0 +1,140 @@
+const Pet = require('../src/pet');
+
+
+describe('constructor', () => {
+    it('returns an object', () => {
+        expect(new Pet('Fido')).toBeInstanceOf(Object);
+    });
+    it('checks initial age of pet', () => {
+        const pet = new Pet('Fido');
+        expect(pet.age).toBe(0);
+    });
+});
+    describe('Declare Pets', () => {
```

Comment:
Each of the describe blocks here should have a clear line before them, as with the `it` blocks. Also your indentation is off here, this describe block should not be indented at all as it is not nested inside anything else. It would be a good idea to make sure your work is all formatted just in general, but especially before submitting for a review